openpower/package/hcode/hcode.mk

```diff
@@ -0,0 +1,50 @@
+################################################################################
+#
+# HCODE
+#
+################################################################################
+
+HCODE_VERSION ?= 270068cbfcb001fa5ccfbb6e8bc18eea22d68487
+HCODE_SITE = $(call github,open-power,hcode,$(HCODE_VERSION))
+
+HCODE_LICENSE = Apache-2.0
+
+HCODE_INSTALL_IMAGES = YES
+HCODE_INSTALL_TARGET = NO
+
+HCODE_DEPENDENCIES = host-binutils host-ppe42-gcc hostboot-binaries
+
+HW_IMAGE_BIN_PATH=output/images/hw_image
+HW_IMAGE_BIN=p9n.hw_image.bin
+HCODE_IMAGE_BIN = p9n.ref_image.bin
+
+CROSS_COMPILER_PATH=$(PPE42_GCC_BIN)
+PPE_TOOL_PATH ?= $(CROSS_COMPILER_PATH)
+PPE_PREFIX    ?= $(PPE_TOOL_PATH)/bin/powerpc-eabi-
```

Comment:
powerpc-eabi- with no other chars is intended? I assume so, just want to make sure.

Reply:
yes, this is just the prefix for finding the tool, it gets pasted on the front of the vars $(CC) $(LD) etc,   which are called powerpc-eabi-cc and so on.

Reply:
I think this is somewhat consistent in how we do other ppe based packages.